### PR TITLE
Rename roles

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: contracts/VoucherHub.sol [hunk 1]
```diff
@@ -25,15 +25,12 @@ contract VoucherHub is
     // Granted to msg.sender == defaultAdmin() == owner()
     // DEFAULT_ADMIN_ROLE
 
-    // TODO: Rename roles.
     // Upgrade VoucherHub and Vouchers contracts.
-    // Granted to msg.sender
```
Comment (gfournieriExec) [inline contracts/VoucherHub.sol:30]: why remove here

**Review by gfournieriExec — APPROVED** [review]
Thanks Next one for Voucher.test.ts ?

File: contracts/VoucherHub.sol [hunk 2]
```diff
@@ -25,15 +25,12 @@ contract VoucherHub is
...
-    bytes32 public constant UPGRADE_MANAGER_ROLE = keccak256("UPGRADE_MANAGER_ROLE");
-    // Add/remove eligible assets.
-    bytes32 public constant ASSET_ELIGIBILITY_MANAGER_ROLE =
-        keccak256("ASSET_ELIGIBILITY_MANAGER_ROLE");
-    // Create & top up Vouchers.
-    bytes32 public constant VOUCHER_MANAGER_ROLE = keccak256("VOUCHER_MANAGER_ROLE");
+    bytes32 public constant UPGRADER_ROLE = keccak256("UPGRADER_ROLE");
+    // Create types required for creating vouchers, add/remove eligible assets, withdraw, [...]
+    bytes32 public constant MANAGER_ROLE = keccak256("MANAGER_ROLE");
+    // Create vouchers & top up vouchers.
```
Comment (zguesmi) [inline contracts/VoucherHub.sol:32]:
```suggestion
    // Create new vouchers & top up existing vouchers.
```
or
```suggestion
    // Create and top up vouchers.
```

File: test/VoucherHub.test.ts [hunk 3]
```diff
@@ -32,23 +32,17 @@ const initVoucherHubBalance = 10n * voucherValue; // arbitrary value, but should
 describe('VoucherHub', function () {
     let iexecPoco: string;
     let iexecPocoInstance: IexecPocoMock;
-    let voucherHubWithVoucherManagerSigner: VoucherHub;
-    let voucherHubWithAssetEligibilityManagerSigner: VoucherHub;
+    let voucherHubWithMinterSigner: VoucherHub;
+    let voucherHubWithManagerSigner: VoucherHub;
```
Comment (zguesmi) [inline test/VoucherHub.test.ts:36]:
```suggestion
    let voucherHubAsMinter: VoucherHub;
    let voucherHubAsManager: VoucherHub;
```
and `voucherHubAsAnyone`

**Comment by jeremyjams** [comment]
> Thanks Next one for Voucher.test.ts ?

@gfournieriExec You are right, here it is [Update role names in Voucher.test.ts](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/29/commits/daf07e2a94995cb96549eff2f4a5a880c3f8f995) !

File: test/VoucherHub.test.ts (hunk 3 above)
Comment (jeremyjams) [reply test/VoucherHub.test.ts:36]: Yes I prefer that too, did not want to change your version :)
I'll do it now

Comment (jeremyjams) [reply test/VoucherHub.test.ts]: See [Update wording to "VoucherHub as role"](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/29/commits/22185d18b3233d185dac9fccc07f3fac1f381384) !

**Review by zguesmi — APPROVED** [review]

**Pull request merged**